BlueprintUI/Tests/StackTests.swift

```diff
@@ -1176,13 +1176,136 @@ class StackTests: XCTestCase {
             ],
             expectedSizes: [
```

Comment:
These changes b/c the flexible elements are now measured in the _available_ space, not _all_ the space.